string topic_name
int8 is_alive
